metecho/api/serializers.py

```diff
@@ -174,15 +176,22 @@ class RepoPermissionSerializer(serializers.Serializer):
 class ShortGitHubUserSerializer(serializers.Serializer):
     """See https://github3py.readthedocs.io/en/master/api-reference/users.html#github3.users.ShortUser"""  # noqa: B950
 
-    id = serializers.CharField(required=False)
+    id = serializers.IntegerField()
```

Comment:
@jerivas The main change that's currently breaking things on the FE is this shift from `github_user.id` being a `string` to an `integer`. I can go through the FE and try to adjust that, but it seems odd that e.g. `user.github_id` is still a `string`. How difficult would it be to standardize this new model `id` as a `string` instead?

Reply:
Turns out `user.github_id` is the outlier here. django-allauth stores it as a string, but the GH API uses an integer. I went ahead and converted `user.github_id` to an int to more closely match the format we're getting from GitHub in dabb10c8459841bdb52901d57b4a3a23ce29a485